web/src/components/gnd-main/index.js

```diff
@@ -16,10 +16,24 @@
  */
 
 import React from "react";
+import { Route } from 'react-router' // react-router v4
 import GndMap from "../gnd-map";
 import GndHeader from "../gnd-header";
 import GndLegend from "../gnd-legend";
 import GndFeatureTypeEditor from "../gnd-feature-type-editor";
+import { connectProject, connectFeature } from "../../datastore.js";
+import { featurePath } from '../../route.js'
+
+
+function GndObservations({ match }) {
+  return (
+    <div>
+      TODO(maralka): exatact observations for feature
```

Comment:
Nit: Please wrap in comment tags `<!--` `-->`

Reply:
Nit: Extract

Reply:
i had to do {/**/}